data/mherzog4.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Matt Herzog",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Engineer",
+  "avatar": "https://github.com/mherzog4.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mattherzog5",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://twitter.com/https://www.linkedin.com/in/mattherzog3/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Visit my website",
+      "url": "http://mattherzog.me/",
+      "icon": "fa-solid fa-code-compare"
```

Comment:
These are not valid icons, please select a valid icon https://linkfree.eddiehub.io/icons